Website/src/rendering/src/components/FullPageSearch/OrderCloudFullPageSearch.tsx

```diff
@@ -0,0 +1,139 @@
+import { useEffect, useState } from 'react';
+import { useRouter } from 'next/router';
+import FullPageSearch, { FullPageSearchResultsProps } from './FullPageSearch';
+import useOcProductList from '../../hooks/useOcProductList';
+import {
+  SearchResultsFacetClickedChangedActionPayload,
+  SearchResultsPageNumberChangedActionPayload,
+  SearchResultsSortChangedActionPayload,
+  SearchResultsKeyphraseChangedActionPayload,
+} from '@sitecore-discover/widgets';
+import { OcProductListOptions, listProducts } from '../../redux/ocProductList';
+import { clone, omit } from 'lodash';
+import { useAppDispatch } from '../../redux/store';
+import { getCategoryIdByUrlPath } from '../../helpers/CategoriesDataHelper';
+
+const OrderCloudFullPageSearch = (): JSX.Element => {
```

Comment:
The OrderCloud version of the full search page seems to have only 24 items to search for while the Discover version has 61 items.

Reply:
I am seeing 61 items on both with no filters applied on this page /shop/products?q=